buildAllSlns.proj

```diff
@@ -1,14 +0,0 @@
-<Project DefaultTargets="Build">
```

Comment:
Why remove it?

Reply:
@jmprieur I put it back. This is due to a duplication of this file that we had on GitHub only and I excluded one of them. However excluding one of them on GitHub caused a deletion on your local repo when you pull the first time from master. You just have to ignore that change once and you are good.